app/src/main/java/ar/scacchipa/twittercloneapp/domain/model/TweetCardInfo.kt

```diff
@@ -0,0 +1,49 @@
+package ar.scacchipa.twittercloneapp.domain.model
+
+data class TweetCardInfo(
```

Comment:
here are all the fields I talk about in the user model, here I think you have to include an object which is the user model that have all the fields (name, image, username, etc etc)

Reply:
Some TweetCardView variables were replaced by a UserInfo variable.